index.html

```diff
@@ -1,122 +1,901 @@
+						fill="#263238" />
+					<path
+						d="M120.373 56.9113C120.312 56.9113 120.312 56.9113 120.373 56.9113C120.068 56.9113 119.886 56.6678 119.947 56.4243C119.947 56.3635 120.007 55.3287 119.034 54.3548C118.181 53.5027 117.086 53.6853 117.025 53.6853C116.903 53.6853 116.842 53.6853 116.721 53.6244C115.868 53.1983 115.808 52.3462 115.929 51.981C115.99 51.7375 116.234 51.6158 116.477 51.6766C116.721 51.7375 116.842 51.981 116.781 52.2245C116.781 52.3462 116.721 52.5288 116.964 52.7114C117.39 52.6505 118.607 52.7114 119.581 53.6853C120.86 54.9635 120.738 56.3635 120.738 56.4243C120.799 56.7287 120.555 56.9113 120.373 56.9113Z"
+						fill="#263238" />
+					<path
+						d="M125.059 74.2585C127.312 75.2933 134.981 77.8497 133.703 81.1365L125.242 106.153L101.869 104.875L106.556 74.6237C107.53 71.8847 109.112 72.0064 113.738 72.189C113.738 72.189 115.016 72.4325 120.312 72.9194C120.373 72.8585 122.99 73.2846 125.059 74.2585Z"
+						fill="#E4594F" />
+					<path
+						d="M117.695 79.98L114.651 72.1281L117.147 72.2499L117.877 68.2935L123.538 69.3282L122.929 73.2846L125.12 74.015L119.094 80.2235C119.034 80.2844 118.242 80.9539 117.695 79.98Z"
+						fill="#FFAB91" />
+					<path
+						d="M90.3042 34.6337C90.1825 34.6337 90.1216 34.5728 90.0607 34.512C89.3303 33.6598 89.3912 32.8077 89.5738 32.2599C88.0521 32.0164 87.5043 30.6164 87.3826 29.886C87.3826 29.7643 87.3826 29.7034 87.4434 29.6425C87.5043 29.5817 87.5652 29.5208 87.6869 29.5208L91.7042 29.2773C91.8868 29.2773 92.0694 29.3991 92.0694 29.5817C92.0694 29.7643 91.9476 29.9469 91.765 29.9469L88.1738 30.1295C88.3565 30.6773 88.8434 31.5903 90.1216 31.5903C90.2433 31.5903 90.3651 31.6512 90.426 31.7729C90.4868 31.8947 90.4868 32.0164 90.3651 32.1381C90.3651 32.199 89.6955 32.9903 90.5477 34.025C90.6694 34.1468 90.6694 34.3902 90.4868 34.512C90.426 34.6337 90.3651 34.6337 90.3042 34.6337Z"
+						fill="#EF6C00" />
+					<path
+						d="M89.0867 30.6773C88.8432 30.6773 88.7215 30.4947 88.7824 30.3121L89.0258 27.7556C89.0258 27.573 89.2084 27.4513 89.391 27.4513C89.5736 27.4513 89.6954 27.6339 89.6954 27.8165L89.4519 30.3729C89.391 30.5555 89.2693 30.6773 89.0867 30.6773Z"
+						fill="#EF6C00" />
+					<path
+						d="M86.8346 33.112C86.652 33.112 86.5303 32.9902 86.5303 32.8076L86.3477 28.4251C86.3477 28.2425 86.4694 28.0599 86.652 28.0599C86.8346 28.0599 87.0172 28.1817 87.0172 28.3643L87.1998 32.7467C87.1998 32.9902 87.0172 33.112 86.8346 33.112Z"
+						fill="#EF6C00" />
+					<path
+						d="M85.252 33.4772C85.0694 33.4772 84.9477 33.3554 84.9477 33.1728L84.3999 29.399C84.3999 29.2164 84.5216 29.0338 84.7042 29.0338C84.8868 29.0338 85.0694 29.1555 85.0694 29.3382L85.6173 33.1119C85.6173 33.2945 85.4955 33.4772 85.252 33.4772Z"
+						
```

Comment:
This &rarr; This request is served from the created Revision

revision &rarr; Revision

service &rarr; Service